betamax/adapter.py

```diff
@@ -96,6 +120,18 @@ def send(self, request, stream=False, timeout=None, verify=True,
 
     def send_and_record(self, request, stream=False, timeout=None,
                         verify=True, cert=None, proxies=None):
+        """Send request and records response.
```

Comment:
"record response" Also this could probably have a paragraph below it describing it's purpose/what it does.